packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -12,31 +12,41 @@ interface IElectionModule {
 
     function getElectionTokenAddress() external view returns (address);
 
-    function getNominees() external view returns (address[] memory);
-
     function nominate() external;
 
     function withdrawNomination() external;
 
-    function setSeatCount(uint seats) external;
-
-    function setPeriodPercent(uint8 percent) external;
-
-    function setNextSeatCount(uint seats) external;
+    function getNominees() external view returns (address[] memory);
 
-    function setNextEpochDuration(uint64 duration) external;
+    function elect(address[] memory candidates) external;
 
-    function setNextPeriodPercent(uint8 percent) external;
+    function isElectionEvaluated() external view returns (bool);
 
-    function elect(address[] memory candidates) external;
+    function evaluateElectionBatch() external;
 
-    function getNomineeVotes(address nominee) external view returns (uint);
+    function resolveElection() external;
 
     function isEpochFinished() external view returns (bool);
 
     function isNominating() external view returns (bool);
 
     function isVoting() external view returns (bool);
 
+    function getSeatCount() external view returns (uint);
```

Comment:
Added getters for the storage variables to be consumed by external operators and the UI